specs/commandline/sign.md

```diff
@@ -19,8 +19,6 @@ Warning: Always sign the artifact using digest(`@sha256:...`) rather than a tag(
 Successfully signed <registry>/<repository>@<digest>
 ```
 
-NOTE: This command is for signing OCI artifacts only. Use `notation blob sign` command for signing arbitrary blobs.
```

Comment:
This part was not removed in `v1.1.1`. Please send out a PR to `release-1.1` to fix it.

Reply:
Tracking: https://github.com/notaryproject/notation/issues/1012